packages/clerk-js/src/core/clerk.ts

```diff
@@ -401,9 +401,11 @@ export class Clerk implements ClerkInterface {
       .then(controls => controls.openModal('userVerification', props || {}));
   };
 
-  public __experimental_closeUserVerification = (): void => {
+  public __experimental_closeUserVerification = (options?: { unstable_notify: boolean }): void => {
```

Comment:
1. We are not expecting people to use `open/closeUserVerification` directly
2. We need `unstable_notify` to be controlled within our util component `UserVerificationModal` in `@shared`